cli/src/hooks/next-check/index.ts

```diff
@@ -0,0 +1,21 @@
+import { Hook } from '@oclif/core';
+import { readFile } from 'fs/promises';
+import path from 'path';
+
+export const getSdkVersion = async (): Promise<null | string> => {
+  const packageJson: any = JSON.parse(await readFile(`${path.join(process.cwd())}/package.json`, 'utf-8'));
+  return packageJson?.dependencies?.['@xata.io/client'] ? packageJson.dependencies['@xata.io/client'] : null;
+};
+
+const hook: Hook<'next-check'> = async function (options: Record<string, unknown>) {
+  if (options.pgrollEnabled) {
+    const sdkVersion = await getSdkVersion();
+    if (sdkVersion && sdkVersion !== 'next') {
```

Comment:
This is what the dep would look like in package.json right? `"@xata.io/client": "next"`

Right now we have alpha versions (`0.0.0-nextv123`), so I could instead check for `-next.v` to cover both cases?

Reply:
Yeah, we should cover alpha versions. The `next` tag will never be there, exact versions are needed.

Reply:
I added a check for `-next.` in the version